evap/staff/importers/enrollment.py

```diff
@@ -608,12 +621,25 @@ def import_enrollments(
             store_participations_in_db(parsed_rows)
 
             msg = format_html(
-                _("Successfully created {} courses/evaluations, {} participants and {} contributors:{}"),
-                new_course_count,
-                new_participants_count,
-                new_responsibles_count,
-                create_user_list_html_string_for_message(new_user_profiles),
+                _("Successfully created {evaluation_string}, {participant_string} and {contributor_string}"),
+                evaluation_string=ngettext(
+                    "1 course/evaluation", "{count} courses/evaluations", new_course_count
+                ).format(count=str(new_course_count)),
+                participant_string=ngettext("1 participant", "{count} participants", new_participants_count).format(
+                    count=str(new_participants_count)
+                ),
+                contributor_string=ngettext("1 contributor", "{count} contributors", new_responsibles_count).format(
+                    count=str(new_responsibles_count)
+                ),
             )
+
+            if new_participants_count or new_responsibles_count:
+                msg = format_html(
+                    "{msg}: {list}", msg=msg, list=create_user_list_html_string_for_message(new_user_profiles)
+                )
```

Comment:
This looks like a copy&paste error: different variables in the if-condition vs in the the `create_user_list` call